demo/lib/TransformationControls/addTextSizeAdjustmentPercentage.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable require-jsdoc */
+
+const textSizeAdjustmentHandler = (iframeWindow, iframeDocument, value) => {
+  if (iframeWindow.pagelib.c1) {
+    iframeWindow.pagelib.c1.PageMods.setTextSizeAdjustmentPercentage(iframeDocument, value)
+  } else {
+    iframeWindow.pagelib.AdjustTextSize.setPercentage(iframeDocument.body, value)
+  }
+}
+
+export default [
+  'adjust_text_size_form',
+  'Text Size Adjustment (Percentage) *',
```

Comment:
maybe add a comment to say that this only works on certain mobile browsers with the caniuse link?

Reply:
You mean a comment in the code or in the sidebar to be showed in the demo interface?

Reply:
Sorry. The latter. I meant to add some user visible text to this input form field. So that it's clearer to demo users why this doesn't work.

Reply:
done.